tools/testsocket/testsocket.cpp

```diff
@@ -1067,6 +1124,16 @@ int main(int argc, char **argv)
             autoXML = false;
             ++arg;
         }
+        else if (strieq(argv[arg], "-gcs"))
+        {
+            generateClientSpans = true;
+            ++arg;
+        }
+        else if (strieq(argv[arg], "-ctc")) 
+        {
+            tracingConfig.set(argv[++arg]);
```

Comment:
good to check if ++arg doesnt >= argc, but we dont check that everywhere above like we should.